### DISCUSSION TITLE
How to implement component into Storybook?

### DISCUSSION THREAD

**Discussion by PhilipZi** [neuefische/questions#88] [Web] [answered]
Guten Tag zusammen, 

ich möchte gerne die Komponente aus dem ersten Bild in Story book einbinden (siehe Bild)
![Screenshot 2022-08-19 at 12 58 25](https://user-images.githubusercontent.com/102238180/185604397-99fd31dc-1941-4e06-b87e-04d82ffcc4e3.png)
)

jetzt wollte ich diese Komponente auf Storybook ansehen und habe in dem terminal npm run storybook eingegeben. darauf hin kam diese Meldung (siehe Bild)
![Screenshot 2022-08-19 at 13 00 12](https://user-images.githubusercontent.com/102238180/185604642-ffe21dd1-e448-4ccc-9cf6-a7e432d64c5a.png)

es muss etwas mit dem svg playbutton zu tun haben. Wie kann ich den generell einsehen im browser auf der Storybook homepage, wie meine stories aussehen? Also wie erhalte ich Zugriff auf die Webseitenansicht?

Vielen Dank im Voraus!

**Comment by ahohnsen** [comment, marked as answer]
Hey Philip,

Das Problem ist, dass Storybook deinen `PlayButton` "nicht erreichen" kann, da dieser im `public` folder liegt. Im `public` folder kannst du Bilddateien (*.svg, *.png, *.jpg etc.) ablegen, aber keine Komponenten. Wenn du deinen `PlayButton` in den `source` folder legst und die Pfade dementsprechend anpasst, dann sollte es funktionieren 🙂 Ich würde dir auch raten, deinen `PlayButton` in z.B. `PlayIcon` umzubennen, da es sich in diesem Fall ja auch nur um ein svg und kein Button-Element handelt.

**Reply by PhilipZi** [reply]
Vielen Dank Andrea! Mega gut!
